### ISSUE TITLE
3.0.1 converge of kibana_lwrp::install fails

### ISSUE THREAD

**Issue by tbunnyman** [lusis/chef-kibana#113]
The end of the run fails when you `include_recipe kibana_lwrp::install` with the following message

```
       [2016-01-25T22:25:42+00:00] ERROR: resource template[/etc/nginx/sites-available/kibana] is configured to notify resource runit_service[nginx] with action reload, but runit_service[nginx] cannot be found in the resource collection. template[/etc/nginx/sites-available/kibana] is defined in /tmp/kitchen/cache/cookbooks/kibana_lwrp/libraries/web.rb:72:in `block (2 levels) in <class:KibanaWeb>'

       [2016-01-25T22:25:42+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

test kitchen
chef 12.6.0
centos 6.7

**Comment by martinb3** [comment]
I also released 3.0.2 maybe 20 minutes ago to fix that other bug in the release. Could you try that or pin back to 2.x for the time being? Thanks!

**Comment by tbunnyman** [comment]
I hadn't gotten the 3.0.2 before running this test. Converge is fixed in 3.0.2

**Comment by martinb3** [comment]
Great! Thanks again for reporting these.
